test/specs/scenarios/accessibility.js

```diff
@@ -0,0 +1,186 @@
+import { localhostUrl } from '../../config.json'
+import { goToPassportUploadScreen, uploadFileAndClickConfirmButton } from './sharedFlows.js'
+import { runAccessibilityTest } from '../../utils/accessibility'
+
+const options = {
+  pageObjects: [
+    'Welcome',
+    'Confirm',
+    'DocumentSelector',
+    'DocumentUpload',
+    'CrossDeviceIntro',
+    'CrossDeviceLink',
+    'CrossDeviceMobileConnected',
+    'CrossDeviceSubmit',
+    'Camera',
+    'LivenessIntro',
+    'PoaDocumentSelection',
+    'PoaGuidance',
+    'PoaIntro',
+    'VerificationComplete',
+    'BasePage'
+  ]
+}
+
+export const accessibilityScenarios = async(lang='en') => {
+
+  describe(`ACCESSIBILITY scenarios in ${lang}`, options, ({driver, pageObjects}) => {
+
+    const {
+      welcome,
+      confirm,
+      documentSelector,
+      documentUpload,
+      crossDeviceIntro,
+      livenessIntro,
+      poaDocumentSelection,
+      poaIntro,
+      basePage
+    } = pageObjects
+
+    const copy = basePage.copy(lang)
+
+    const goToPoADocumentSelectionScreen = async () => {
+      driver.get(localhostUrl + `?poa=true&async=false&useWebcam=false`)
+      welcome.primaryBtn.click()
+      poaIntro.clickStartVerificationButton()
+    }
+
+    const goToCrossDeviceScreen = async () => {
+      welcome.primaryBtn.click()
+      documentSelector.passportIcon.click()
+      documentUpload.crossDeviceIcon.click()
+      crossDeviceIntro.continueButton.click()
+    }
+    
+    //Welcome
+    it('should verify accessibility for the welcome screen', async () => {
+      driver.get(`${localhostUrl}?language=${lang}`)
+      runAccessibilityTest(driver)
+    })
+
+    it('should verify focus management for the welcome screen', async () => {
+      driver.get(`${localhostUrl}?language=${lang}`)
+      welcome.verifyFocusManagement()
+    })
+
+    //Cross Device Sync
+    it('should verify accessibility for the cross device intro screen', async () => {
+      driver.get(`${localhostUrl}?language=${lang}`)
+      welcome.primaryBtn.click()
+      documentSelector.passportIcon.click()
+      documentUpload.crossDeviceIcon.click()
+      runAccessibilityTest(driver)
+    })
+
+    it('should verify accessibility for the cross device screen', async () => {
+      driver.get(`${localhostUrl}?language=${lang}`)
+      goToCrossDeviceScreen()
+      runAccessibilityTest(driver)
+    })
+
+    // Document Selector
+    it('should verify accessibility for the document selector screen', async () => {
+      driver.get(`${localhostUrl}?language=${lang}`)
+      welcome.primaryBtn.click()
+      runAccessibilityTest(driver)
+    })
+
+    //Document Upload
+    it('should verify accessibility for the uploader screen', async () => {
+      goToPassportUploadScreen(driver, welcome, documentSelector, `?language=${lang}`)
+      runAccessibilityTest(driver)
+    })
+
+    //FIXME: This is commented out due to the accessibility violations we need to fix.
```

Comment:
`document-title` should be added to the `disabledAccessibilityRules` list

Reply:
I added that and ran test again, now this rule fails `aria-valid-attr-value`

```AssertionError: 1 violations found:
- Rule ID: aria-valid-attr-value
  ARIA attributes must conform to valid values (1 elements affected)
  Help: https://dequeuniversity.com/rules/axe/3.3/aria-valid-attr-value?application=webdriverjs
: expected 1 to equal 0
```

Reply:
Done.
5549af7